src/lib/markers/public/TimelineMarkers.js

```diff
@@ -0,0 +1,20 @@
+// Is this necessary? The initial reason for including this is for organization sake in the
+// user code e.g.
```

Comment:
I really like that you decided to include this. This makes the API more declarative or at least that is what I think

Reply:
I agree that its more declarative and I could see this component taking props in the future. Just don't tell anyone these markers without using this component :)